lib/issue-comment-handler.js

```diff
@@ -58,6 +60,10 @@ function issueCommentHandler(event) {
     if (!which('git')) {
       return Promise.reject(new Error('Sorry, this script requires git'));
     }
+    if (exec('git config --get user.email')) {
+      exec('git config --global user.name "seanbot"');
+      exec('git config --global user.email sean.lee@theweflex.com');
```

Comment:
developer@theweflex.com looks like a group mail list，so I prefer seanbot@theweflex.com.
and we need to edit the email address at seanbot github account